fixbackend/dispatcher/dispatcher_service.py

```diff
@@ -149,6 +223,20 @@ async def compute_next_run(self, tenant: WorkspaceId, last_run: Optional[datetim
         log.info(f"Next run for tenant: {tenant} is {result}")
         return result
 
+    async def _add_collect_in_progress_account(
+        self, workspace_id: WorkspaceId, job_id: str, account_id: CloudAccountId
+    ) -> None:
+        value = AccountCollectInProgress(job_id, str(account_id), utc().isoformat()).to_json_str()
```

Comment:
I wonder if we need to handle the case, where InProgress is present (and not done).
Should we even trigger a new run?
Maybe better: reduce the expiration time from 24h to 3hours. Do not trigger a new collect cycle if not all jobs are done. If a job died or a message did not arrive, we will trigger a new run after 3h.
We only have a problem if there are collect runs that last longer than 3 hours - but I assume that's something we should handle anyway. wdyt?